lib/ExtUtils/MM_Haiku.pm

```diff
@@ -0,0 +1,81 @@
+package ExtUtils::MM_Haiku;
+
+use strict;
+
+=head1 NAME
+
+ExtUtils::MM_Haiku - methods to override UN*X behaviour in ExtUtils::MakeMaker
+
+=head1 SYNOPSIS
+
+ use ExtUtils::MM_Haiku;    # Done internally by ExtUtils::MakeMaker if needed
+
+=head1 DESCRIPTION
+
+See ExtUtils::MM_Unix for a documentation of the methods provided
+there. This package overrides the implementation of these methods, not
+the semantics.
+
+=over 4
+
+=cut
+
+use ExtUtils::MakeMaker::Config;
+use File::Spec;
+require ExtUtils::MM_Any;
```

Comment:
These two can just be a `use ExtUtils::MM_Unix;`

Reply:
I fixed it, but notes in my update I kept the `require` instead of `use` (is it good?)

Reply:
It's not wrong, just a little silly